train.py

```diff
@@ -27,7 +27,7 @@
 WAVENET_PARAMS = './wavenet_params.json'
 STARTED_DATESTRING = "{0:%Y-%m-%dT%H-%M-%S}".format(datetime.now())
 SAMPLE_SIZE = 100000
-
+L2_REGULARIZATION_STRENGTH=0.
```

Comment:
I can imagine that users might want to stop the training and restart it with a different regularization parameter. Although this would also work by editing `wavenet_params.json`, I think that having it as part of the training script makes things easier.
It also seems natural to have it next to the learning rate.